LABELS.md

```diff
@@ -0,0 +1,64 @@
+# GitHub Labels
+
+Simple Icons uses labels extensively to categorize issues and contributions. This document aims to explain the labels as well as define rules around how labels are used.
+
+## Notes
+
+- We list various time windows here. If you need an extension for your contribution, you can indicate the time you need in the issue or Pull Request subject to the time window. If the request is reasonable, the deadlines in this document are applied **after** the deadline you set for yourself.
+- If you see an issue or Pull Request that exceeded the time windows listed here, feel free to leave a comment reminding the @simple-icons/maintainers to take action.
+
+## Labels
+
+### Good First Issue
+
+The `good first issue` label is intended to make it easier for newcomers to the project to make a first contributions. Issues with this label are expected to not require too much work, allowing newcomers to get familiar with the contribution process without having to worry too much about technical details.
+
+For the first **14 days** after being assigned to an issue, the issue should not be picked up by existing contributors or maintainers. After that, anyone can work on the issue. The `good first issue` label will remain on the issue until it is picked up by someone.
```

Comment:
We, now, ask users if they want to contribute in our issue forms:
![image](https://user-images.githubusercontent.com/12817388/135857730-0bdd9b9b-6944-4bf7-8a8d-c03bb1523aa9.png)

If they answer "Yes", I'm wondering if we should have the same rule applied. Wait at least 14 days to let the users contribute themselves.